### DISCUSSION TITLE
Improve packages list

### DISCUSSION THREAD

**Discussion by yairEO** [npm/feedback#282] [Website] [answered]
### Currently the design of the *packages list* is almost identical across the website: 
viewing my own (or org) packages, another person's (or org) packages or even the list on the homepage.

Those list provide little beneficial information that can help making a decisions when comparing between packages, and in my case, as an author or multiple packages, I would like to see the stats of all my packages right there in the packages page:

![image](https://user-images.githubusercontent.com/845031/111062295-b292ae80-84b0-11eb-8d63-9ad29d4767a0.png)

**This page has so much more it can show, and has plenty of whitespace.**

I would love to see it as a table with important stats which helps me see the ***big picture*** of my whole portfolio of packages:

First, a column for `Weekly Downloads` would be helpful, with a number and small chart.
Also helpful is the number of issues, which helps me understand other packages, since it's a crucial measurement of maintenance, for example, I would very much hesitate using a package with hundreds of open issues unless I have no choice.
Also, a package with 0 open issues can be suspicious in accordance to other parameters. 

 The `size` is also extremely important to show when listing packages. 

Basically almost everything which can be viewed on a single-package page is highly useful also on packages listing page:

![image](https://user-images.githubusercontent.com/845031/111062466-9e9b7c80-84b1-11eb-9562-037af7b1b82c.png)

**Comment by ljharb** [comment]
Why is the size important, especially when viewing your own org?

Having many open issues is not, in fact, an accurate measure of maintenance - plenty of unmaintained packages have issues disabled, or use a bot to auto-close issues; and plenty of well-maintained packages leave issues open on purpose as a todo list rather than closing them and pretending there’s nothing wrong.

**Reply by yairEO** [reply]
It matters. I sometimes add too many files or forget when I publish and not all of my packages have failsafe scripts for that. it's helpful information information 

Also to keep the UI of the website consistent when the list shows my own repos or others'


But first I would really want to see the download stats

**Comment by MylesBorins** [comment, marked as answer]
Hey @yairEO thanks for the suggestion. This is definitely a great improvement that we could make the the npmjs.com website, but I want to set expectations that we are not currently prioritizing front-end work like this for the next quarter. We are in the process of planning right now and will be updating the [roadmap](https://github.com/npm/roadmap) once we've locked in the work we'll be doing.

I've added a note in the roadmap to future work so we won't lose this suggestion, but will hold off on expanding it into a full tracking issue until we have a better idea of when we could prioritize this work.

**Reply by yairEO** [reply]
Thanks for the update. 
I am curious - How many front-end developers work on **npmjs**?

**Reply by MylesBorins** [reply]
@yairEO engineering is not split up that way. We have teams that get moved around to different initiatives based on our overall objectives + roadmap.
